RecoEgamma/EgammaIsolationAlgos/interface/EleTkIsolFromCands.h

```diff
@@ -100,40 +103,37 @@ class EleTkIsolFromCands {
   // preselected by the cuts that can already be applied without considering
   // the electron. Note that this has to be done twice, because the required
   // preselection is different for barrel and endcap electrons.
-  //
-  class SimpleTrack {
-  public:
-    SimpleTrack(reco::TrackBase const& trk) : pt{trk.pt()}, eta{trk.eta()}, phi{trk.phi()}, vz{trk.vz()} {}
-    const double pt;
-    const double eta;
-    const double phi;
-    const double vz;
-  };
+
+  SOA_DECLARE_COLUMN(Pt, double, "pt");
+  SOA_DECLARE_COLUMN(Eta, double, "eta");
```

Comment:
in my opinion float will suffice

Reply:
Thanks for the comment, I'll move all computations to float.